platform/src/main/java/org/stellar/anchor/platform/config/PropertyClientsConfig.java

```diff
@@ -37,82 +33,85 @@ public boolean supports(@NotNull Class<?> clazz) {
 
   @Override
   public void validate(@NotNull Object target, @NotNull Errors errors) {
-    if (type.equals(CLIENTS_CONFIG_TYPE_FILE)) {
-      // validate file type
-      Map<String, Object> map = emptyMap();
+    // If type is FILE, value must be defined
+    // if type is INLINE, items can be left empty as this not a required config
+    if (this.getType() == ClientsConfigType.FILE && isEmpty(this.getValue())) {
+      errors.reject("invalid-no-value-defined", "clients.value is empty. Please define.");
+    }
+
+    // Parse the file and validate the contents
+    if (type.equals(ClientsConfigType.FILE)) {
       try {
-        map = parseFileToMap(this.getValue());
+        items = parseFileToList(this.getValue());
       } catch (InvalidConfigException e) {
         error("Error loading clients file", e);
         errors.reject(
             "clients-file-not-valid", "Cannot read from clients file: " + this.getValue());
       }
-      // validate file content
-      validateCustodialClients(getCustodialClientsFromMap(map), errors);
-      validateNonCustodialClients(getNonCustodialClientsFromMap(map), errors);
-    } else {
-      // validate inline config
-      validateCustodialClients(custodial, errors);
-      validateNonCustodialClients(noncustodial, errors);
     }
-  }
 
-  void validateCustodialClients(List<CustodialClientConfig> clients, Errors errors) {
-    for (CustodialClientConfig client : clients) {
-      debugF("Validating custodial client {}", client);
-      if (client.getSigningKeys() == null || client.getSigningKeys().isEmpty()) {
+    // validate custodial client and noncustodial client
+    for (TempClient item : items) {
+      if (item.getType().equals(ClientConfig.ClientType.CUSTODIAL)) {
```

Comment:
If `item.getType` is null, this will throw an NPE. 

We could change this to the following to get around this:
```java
ClientConfig.ClientType.CUSTODIAL.equals(item.getType())
```

But should we validate that all clients have a `type` field before validating them instead?